data/person/01K5ST4B1SH3CZ9RGMTJRWHP4K.toml

```diff
@@ -18,12 +18,3 @@ type = "congress"
 congress = 16
 position = "representative"
 
-[[memberships]]
```

Comment:
Should be added here:

https://en.wikipedia.org/wiki/17th_Congress_of_the_Philippines
https://en.wikipedia.org/wiki/18th_Congress_of_the_Philippines